tests/e2e/util/commonUtils.go

```diff
@@ -73,11 +73,11 @@ func Shell(format string, args ...interface{}) (string, error) {
 	glog.V(2).Infof("Running command %s", command)
 	c := exec.Command(parts[0], parts[1:]...) // #nosec
 	bytes, err := c.CombinedOutput()
-	glog.V(2).Infof("Command output: \n %s, err: %v", string(bytes[:]), err)
 	if err != nil {
 		return string(bytes), fmt.Errorf("command failed: %q %v", string(bytes), err)
 	}
 
+	glog.V(2).Infof("Command output: \n %s", string(bytes[:]))
```

Comment:
I wonder if we need a "ShellDontLogErrors" instead or add a bool error ok in the signature
it's a bit strange that at V2 we wouldn't log at all the output
maybe this should be V3+ and the callers should log